### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by MaedahBatool**
Hey,
There were some punctuation and grammatical mistakes which I corrected. Kindly review them. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ Printing since 1440.
 
 ![Text block](https://wpcoredesign.mystagingwebsite.com/gutenberg/gutenberg.png)
 
-This is the development hub for the <a href="https://make.wordpress.org/core/2017/01/04/focus-tech-and-design-leads/">editor focus in core</a>. Gutenberg is the project name. If you want the latest release for your WordPress, <a href="https://wordpress.org/plugins/gutenberg/">download Gutenberg from the WordPress.org plugins repository</a>. Conversations and discussions take place in #core-editor on <a href="https://make.wordpress.org/chat/">the core WordPress Slack</a>. This is our kickoff goal:
+This is the development hub for the <a href="https://make.wordpress.org/core/2017/01/04/focus-tech-and-design-leads/">editor focus in the core</a>. Gutenberg is the project name. If you want the latest release for your WordPress, <a href="https://wordpress.org/plugins/gutenberg/">download Gutenberg from the WordPress.org plugins repository</a>. Conversations and discussions take place in #core-editor on <a href="https://make.wordpress.org/chat/">the core WordPress Slack</a>. This is our kickoff goal:
```
Comment (nylen) [inline README.md]: This is correct as it's currently written. "in core" means "in WordPress core".

File: README.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ Printing since 1440.
...
 
-> The editor will endeavour to create a new page and post building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery. — Matt Mullenweg
+> The editor will endeavor to create a new page and post building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery. — Matt Mullenweg
```
Comment (nylen) [inline README.md]: This is the British spelling. It's fine as currently written.

File: README.md [hunk 3]
```diff
@@ -24,15 +24,15 @@ Gutenberg is built by many contributors and volunteers. Please see the full list
 - <a href="https://make.wordpress.org/core/2017/05/05/editor-how-little-blocks-work/">How Blocks Work</a>
 - <a href="https://github.com/Automattic/wp-post-grammar">WP Post Grammar Parser</a>
 - <a href="https://make.wordpress.org/core/tag/gutenberg/">Development updates on make.wordpress.org</a>
-- <a href="http://gutenberg-devdoc.surge.sh/">Documentation: Creating Blocks, Reference and Guidelines</a>
+- <a href="http://gutenberg-devdoc.surge.sh/">Documentation: Creating Blocks, Reference, and Guidelines</a>
```
Comment (nylen) [inline README.md]: This is an Oxford comma. It's optional, so it's fine as currently written.

File: README.md [hunk 4]
```diff
@@ -24,15 +24,15 @@ Gutenberg is built by many contributors and volunteers. Please see the full list
...
 
 ## How You Can Contribute
 
 Please see [CONTRIBUTING.md](CONTRIBUTING.md).
 
 ## Why
 
-One thing that sets WordPress apart from other systems is that it allows you to create as rich a post layout as you can imagine -- but only if you know HTML & CSS and build your own custom theme. By thinking of the editor as a tool to let you write rich posts, and in a few clicks create beautiful layouts, hopefully, we can make people start to _love_ WordPress, as opposed to pick it because it's what everyone else uses to blog.
+One thing that sets WordPress apart from other systems is that it allows you to create as rich a post layout as you can imagine -- but only if you know HTML & CSS and build your own custom theme. By thinking of the editor as a tool to let you write rich posts, and in a few clicks create beautiful layouts. Hopefully, we can make people start to _love_ WordPress, as opposed to picking it because it's what everyone else uses to blog.
```
Comment (nylen) [inline README.md]: :+1:

File: README.md [hunk 5]
```diff
@@ -46,4 +46,4 @@ By showing critical UI in the body of the content, many can get their basic blog
 
 When the Post Settings sidebar is open — which it is by default — you are essentially in advanced layout mode. By default, you'll see all your metaboxes right there.
 
-Every block can be _inspected_ by clicking it. And every block has advanced layout options available in the inspector; text might have drop-cap, image might have fixed position scrolling. As such, block attributes fall in two camps — the most important ones available right on the block, advanced ones living in the sidebar inspector.
+Every block can be _inspected_ by clicking it. And every block has advanced layout options available in the inspector; text might have drop-cap, the image might have fixed position scrolling. As such, block attributes fall into two camps — the most important ones available right on the block, advanced ones living in the sidebar inspector.
```
Comment (nylen) [inline README.md]: "the image" - `image` refers to a specific block type, same as `text`. We can put backticks `\`` around them but they are correct as currently written.

"into" - :+1:

**Review by nylen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MaedahBatool) [inline README.md]: Okay!

File: README.md (hunk 2 above)
Comment (MaedahBatool) [inline README.md]: 👍

File: README.md (hunk 3 above)
Comment (MaedahBatool) [inline README.md]: 👍

File: README.md (hunk 1 above)
Comment (aduth) [inline README.md]: We could be more explicit about "in WordPress core" if it's likely to cause confusion.

**Comment by gziolo** [comment]
@MaedahBatool do you plan to fix the remaining issues?

**Comment by karmatosed** [comment]
Closing as no movement on this for a while. We can reopen if someone wants to pick up the changes.

**Pull request closed without merging**